### ISSUE TITLE
When pseudoTrans=true, and picom is not in use, and movie is being played, paging gives bad flicker

### ISSUE THREAD

**Issue by felixfung** [felixfung/skippy-xd#84]

**Comment by felixfung** [comment]
With PR #86 I tidied up some related code, and (re)gained some understanding around the logic of paging redraw.

Looks like to me that without picom or similar compositor, when `pseudoTrans = true`, the only way to redraw a paging desktop is with a sequence of unmapping, redrawing, remapping the paging desktop window, which on high frequency refreshes (e.g. when a movie is being played) leads to flickers.

Even when the redrawing and unmapping/remapping frequency is throttled to the point the movie becomes unwatchable, the flicker remains.

What comes to mind is some sort of double buffering to reduce or eliminate the flicker. My understanding of the XComposite library and the object lifecycles of related variables is that the way to implement this will be super involved, takes a prohibitive amount of CPU cycles, or downright impossible within the current scope of skippy-xd.

In other words, the flickers can be interpreted as skippy-xd hitting the limitation of XComposite, at least based on my understanding of one, or the other, or both. Hence, I am inclined to not deal with this issue. However, I am going to keep this open, so that others will see that this issue is known. If others can fix/improve this, all the better.
